src/main/resources/templates/home.html

```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
```

Comment:
Your templates needed some work. I added the layout header in the html tag to get a more consistent look.